### PR TITLE
Setting snappy dependency to 1.1.1.7 since 1.0.5 causes problems with Kafka RealtimeSource

### PR DESCRIPTION
**Pull request by gokulavasan**
Bamboo : http://builds.cask.co/browse/CDAP-RT23-1

### CODE REVIEW

**Comment by henry-cask** [comment]
+1
LGTM

File: cdap-app-templates/cdap-etl/cdap-etl-lib/pom.xml [hunk 1]
```diff
@@ -179,6 +179,10 @@
       <groupId>org.apache.kafka</groupId>
       <artifactId>kafka_2.10</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.xerial.snappy</groupId>
+      <artifactId>snappy-java</artifactId>
```
Comment (henry-cask) [inline cdap-app-templates/cdap-etl/cdap-etl-lib/pom.xml]: I remember ordering of dependency matter in certain maven version. Probably should add this dependency before Kafka just to be safe?

Comment (chtyim) [inline cdap-app-templates/cdap-etl/cdap-etl-lib/pom.xml]: Better be exclude the transitive one from Kafka. Maven now depends on the depth of the dependency tree to determine the version

Comment (henry-cask) [inline cdap-app-templates/cdap-etl/cdap-etl-lib/pom.xml]: Good idea,

**Comment by gokulavasan** [comment]
@chtyim Excluded snappy dep in etl-lib pom. Should this be done in cdap pom instead?

**Comment by chtyim** [comment]
LGTM.

**Pull request merged**